src/identity/components/GlobalHeader/IdentityUserAvatar.tsx

```diff
@@ -94,22 +94,24 @@ class IdentityUserAvatar extends React.Component<Props, State> {
       'user-popover--open': this.state.isPopoverOpen,
     })
 
+    const avatarInitials = this.getInitials()
+
     const {isPopoverOpen} = this.state
     return (
       <ClickOutside onClickOutside={this.setPopoverStateClosed}>
         <div>
           {/* Button shape is ButtonShape.Square to make the height and width the same
             so we can use the border radius to make it a circle  */}
           <Button
-            text={this.getInitials()}
+            text={avatarInitials}
             shape={ButtonShape.Square}
             color={
               isPopoverOpen ? ComponentColor.Default : ComponentColor.Tertiary
             }
             onClick={this.togglePopoverState}
             className={userAvatarButtonClassName}
             testID="global-header--user-avatar"
-            icon={!this.getInitials() ? IconFont.User : null}
+            icon={avatarInitials === '' ? IconFont.User : null}
```

Comment:
This is to be a little more explicit about what we expect here. Since `firstName` and `lastName` are forced to be strings, the only case where we render `IconFont.User` - the default profile image that doesn't have user initials - is when both names are empty strings.